markdown-badges.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+ci_root=$(dirname "$0")
+
+echo "Copy and paste the output below into README.md:"
+echo
+
+for workflow in "$ci_root"/.github/workflows/*.yml; do
```

Comment:
is there a way to do a numeric sort? seeing 4.14 first and the resulting ordering is funny

Reply:
Which would look like this, if that is better to you?

[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/tip/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Atip)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/next/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Anext)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/mainline/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Amainline)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/arm64/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aarm64)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/arm64-fixes/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aarm64-fixes)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-mainline/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-mainline)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-4.9/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-4.9)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-4.19/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-4.19)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-4.14/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-4.14)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android13-5.15/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid13-5.15)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android13-5.10/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid13-5.10)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android12-5.4/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid12-5.4)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android12-5.10/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid12-5.10)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/5.4/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A5.4)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/5.15/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A5.15)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/5.10/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A5.10)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/4.9/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A4.9)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/4.19/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A4.19)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/4.14/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A4.14)

Holy red, Batman!

Reply:
isn't there a `sort -n` flag for numeric sort? that should put 4.9 AFTER 4.19 for example

Reply:
This is what `sort -n` generates:

[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/4.14/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A4.14)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/4.19/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A4.19)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/4.9/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A4.9)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/5.10/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A5.10)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/5.15/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A5.15)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/5.4/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3A5.4)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android12-5.10/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid12-5.10)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android12-5.4/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid12-5.4)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android13-5.10/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid13-5.10)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android13-5.15/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid13-5.15)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-4.14/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-4.14)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-4.19/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-4.19)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-4.9/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-4.9)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/android-mainline/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aandroid-mainline)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/arm64-fixes/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aarm64-fixes)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/arm64/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Aarm64)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/mainline/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Amainline)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/next/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Anext)
[![Actions Status](https://github.com/clangbuiltlinux/continuous-integration2/workflows/tip/badge.svg)](https://github.com/clangbuiltlinux/continuous-integration2/actions?query=workflow%3Atip)

Reply:
That generates the exact same output as the second comment (so no change from just `sort -n`).

Reply:
how about `sort -n --field-separator=. --key=1,1n --key=2,2n`?

Reply:
Nope, same as above. Perhaps I could write this in Python and we could add an `order` or `priority` key to the tree items in `generator.yml` so that we control the exact order they are showed in?

Reply:
it's fine for now. Maybe a nice to have for another day. Filed https://github.com/ClangBuiltLinux/continuous-integration2/issues/303 to follow up on. Not necessary for landing this PR.

Reply:
Sounds good, I will just keep it as it is for this PR and I'll see if I can do something like I described above when it is less busy.